### PR TITLE
Create a fling-based minigame

### PR DESCRIPTION
**Pull request by josephkimsungha**
Minigame overview: 3 squares are rendered and need to be flung off the screen to win

### CODE REVIEW

File: src/minigames/flingMinigame.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+  private handlePhysics(time: Ticker, object: Container) {
+    if (this.dragTarget === object) return;
+
+    const vel = this.velocityInfo[object.label];
+    const adjustedVel = vel;
+    adjustedVel.x = vel.x * time.deltaTime;
+    adjustedVel.y = vel.y * time.deltaTime;
+
+    // object.position.add(vel.multiplyScalar(time.deltaTime));
```
Comment (zainafzal08) [inline src/minigames/flingMinigame.ts]: nit: remove comments.

File: src/minigames/flingMinigame.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    this.dragTarget = undefined;
+  }
+
+  private onDragMove(event: FederatedMouseEvent) {
+    if (!this.dragTarget) return;
+
+    const previousPos = this.dragTarget.position.clone();
+    this.dragTarget.position = event.getLocalPosition(this.dragTarget.parent);
+
+    // this.dragTargetVelocity = this.dragTarget.position.subtract(previousPos);
```
Comment (zainafzal08) [inline src/minigames/flingMinigame.ts]: nit: remove

**Review by zainafzal08 — APPROVED** [review]
Seems like a good starting point. We can fix some of the bugs in follow ups

File: src/minigames/flingMinigame.ts (hunk 1 above)
Comment (josephkimsungha) [reply src/minigames/flingMinigame.ts]: Figured out the side effect imports so using the `.add()` approach now.

File: src/minigames/flingMinigame.ts (hunk 2 above)
Comment (josephkimsungha) [reply src/minigames/flingMinigame.ts]: ditto

**Pull request merged**